build/pause/Makefile

```diff
@@ -17,15 +17,15 @@
 REGISTRY ?= staging-k8s.gcr.io
 IMAGE = $(REGISTRY)/pause
 
-TAG ?= 3.9
+TAG ?= 3.10
 REV = $(shell git describe --contains --always --match='v*')
 
 # Architectures supported: amd64, arm, arm64, ppc64le and s390x
 ARCH ?= amd64
 # Operating systems supported: linux, windows
 OS ?= linux
-# OS Version for the Windows images: 1809, ltsc2022
-OSVERSION ?= 1809 ltsc2022
+# OS Version for the Windows images: 1809 (deprecating), ltsc2019, ltsc2022
+OSVERSION ?= 1809 ltsc2019 ltsc2022
```

Comment:
So, 1809 and ltsc2019 have the exact same ``os.version`` field:

```
ubuntu@ubuntu:~/workdir/kubernetes$ docker manifest inspect mcr.microsoft.com/windows/nanoserver:ltsc2019
{
   "schemaVersion": 2,
   "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
   "manifests": [
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 574,
         "digest": "sha256:5b4a28560f4b658d2154bc2d269c4833c36205644d3ffe52b107950ed12ecc83",
         "platform": {
            "architecture": "amd64",
            "os": "windows",
            "os.version": "10.0.17763.4252"
         }
      }
   ]
}
ubuntu@ubuntu:~/workdir/kubernetes$ docker manifest inspect mcr.microsoft.com/windows/nanoserver:1809
{
   "schemaVersion": 2,
   "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
   "manifests": [
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 574,
         "digest": "sha256:5b4a28560f4b658d2154bc2d269c4833c36205644d3ffe52b107950ed12ecc83",
         "platform": {
            "architecture": "amd64",
            "os": "windows",
            "os.version": "10.0.17763.4252"
         }
      }
   ]
}
```

And they have same digest. So, they're the same image under different tags.

I think we could simply replace 1809 with ltsc2019 directly. In fact, I'm not 100% sure it's alright for us to create a manifest list in which we have 2 identical images, with the same os, arch, and os.version fields. I wonder if that can cause an issue while fetching (although it shouldn't).

Reply:
I think we can keep both for compatibility.